plugins/wporg-5ftf/includes/contributor.php

```diff
@@ -68,6 +71,25 @@ function register_custom_post_type() {
 	register_post_type( CPT_ID, $args );
 }
 
+/**
+ * Register the post status(es).
+ *
+ * @return void
+ */
+function register_custom_post_status() {
+	register_post_status(
+		FiveForTheFuture\PREFIX . '-deactivated',
+		array(
+			'label'       => __( 'Deactivated', 'wporg' ),
+			'label_count' => _n_noop( 'Deactivated <span class="count">(%s)</span>', 'Deactivated <span class="count">(%s)</span>', 'wporg' ),
+			'public'      => false,
+			'internal'    => false,
+			'protected'   => true,
+			CPT_ID        => true, // Custom parameter to streamline its use with the Contributor CPT.
+		)
+	);
+}
```

Comment:
Per my comment [here](https://github.com/WordPress/five-for-the-future/issues/59#issuecomment-547028051), I'm not convinced we need a custom status here. There isn't currently a workflow where we would reactivate a contributor post that had previously been deactivated.

For record keeping purposes, there's [this PR](https://github.com/WordPress/five-for-the-future/pull/54), which would keep track of when each contributor was added and removed from a pledge.

Reply:
Sounds good, I'll remove this.